### PR TITLE
Deduped realtime collections that are also stored.

### LINKED ISSUE #67: Duplicate collection in treeview storage/realtime
How to reproduce?

* go to http://localhost:3000/#!/data/kuzzle-bo-testindex/editable-collection/watch
* click on subscribe
* open an other window on  http://localhost:3000/#!/data/kuzzle-bo-testindex/

Collection `editable-collection` is now in storage and in realtime. We have to exclude from realtime collections already in present in storage.

### PR DESCRIPTION
**Pull request by xbill82**
Fixes #67

### CODE REVIEW

**Comment by AnthonySendra** [comment]
:+1:

File: src/vuex/modules/data/actions.js [hunk 1]
```diff
@@ -45,6 +46,7 @@ export const listIndexesAndCollections = (store) => {
             }
             if (index !== '%kuzzle') {
               addLocalRealtimeCollections(result, index)
+              result = dedupeRealtimeCollections(result)
```
Comment (stafyniaksacha) [inline src/vuex/modules/data/actions.js]: Hum, 2 point here: 
- `dedupeRealtimeCollection()` should be called after each call of `addLocalRealtimeCollections()` -> so you've missed a [call at line 87](https://github.com/kuzzleio/kuzzle-backoffice/blob/627a7ae802858e696d764d93203c52c385b5ea1a/src/vuex/modules/data/actions.js#L87)
- OR `dedupeRealtimeCollection()` logic should be added into `addLocalRealtimeCollections()`

What did you think ?

Comment (stafyniaksacha) [inline src/vuex/modules/data/actions.js]: Ok, didn't see that you have removed this part in #71 !

**Comment by stafyniaksacha** [comment]
+1

**Pull request merged**